src/bundles/SellerRegistration/StartWidget.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import RegisterComponent from '../../RegisterComponent'
+
+export const StartWidget = (props) => {
+    return (
```

Comment:
Since this is a presentational component, you can simplify this.

from:
```javascript
export const StartWidget = (props) => {
  return (
    <div>
      ...
    </div>
  )
}
```
to:
```javascript
export const StartWidget = (props) => (
  <div>
    ...
  </div>
)
```

This is fine for now, but this might be helpful in the future.